app/terminal/terminal.tsx

```diff
@@ -66,14 +63,15 @@ export default class TerminalComponent extends React.Component<TerminalProps, Te
 
   handleWrapClicked() {
     let shouldWrap = !this.state.wrap;
-    localStorage.setItem(wrapLocalStorageKey, shouldWrap ? wrapLocalStorageValue : undefined);
+    localStorage.setItem(WRAP_LOCAL_STORAGE_KEY, shouldWrap ? WRAP_LOCAL_STORAGE_VALUE : undefined);
     this.setState({ wrap: shouldWrap });
   }
 
   handleDownloadClicked() {
     var element = document.createElement("a");
-    element.setAttribute("href", "data:text/plain;charset=utf-8," + encodeURIComponent(this.props.value));
-    element.setAttribute("download", "build_logs.txt");
+    const unstyledLogs = this.props.value.replace(ANSI_STYLES_REGEX, "");
+    element.setAttribute("href", "data:text/plain;charset=utf-8," + encodeURIComponent(unstyledLogs));
+    element.setAttribute("download", "build.log");
```

Comment:
i'm kinda partial to .txt but ¯\\\_(ツ)\_/¯

Reply:
Do you mean you prefer the `.txt` extension itself, or the plain-text rendering style of `.txt` vs `.log`? Here's a comparison of how VS Code handles the two different file types -- I think it does a decent job of improving the signal-to-noise ratio (some other editors might implement this as well):

![image](https://user-images.githubusercontent.com/2414826/141514654-d6b8ad55-2da6-4772-a0c0-a1e5680d9157.png)

Reply:
I base this mostly on my own experience, but it seems like .txt is more widely supported / understood. Pretty mild preference here though, I feel like .log is easily figure-out-able, but it's annoying when it doesn't work automatically.

Agree it looks nicer when it does work.

Are there any numbers about this? How confident are you that everyone can open .log files?

Reply:
I will revert this for now though until I'm able to test on more machines. The more important change is stripping out the ANSI styles so that the text doesn't look messed up when opened with the user's preferred txt viewer.